### PR TITLE
added katonic ai

### PR DESCRIPTION
**Pull request by rahul-katonic**
Please merge this PR

### CODE REVIEW

File: data.yml [hunk 1]
```diff
@@ -207,6 +207,16 @@ landscape:
             twitter: https://twitter.com/AirtrainAI
             crunchbase: https://www.crunchbase.com/organization/sematic-2bbe
             logo: airtrain.svg
+      - subcategory:
+            name: Generative AI Platform
```
Comment (gaocegege) [inline data.yml]: The term "Generative AI Platform" is too broad for my understanding. Could you please provide a more specific context or description?

Comment (rahul-katonic) [reply data.yml]: we are an end to end platform company like Google Gen AI studio and fall under multiple sub categories. what would be the right term in this case?

Comment (gaocegege) [reply data.yml]: How about end-to-end platform?

Comment (rahul-katonic) [reply data.yml]: yes, this would work

Comment (rahul-katonic) [reply data.yml]: Should I create a new PR, with end-to-end subcategory?

**Comment by rahul-katonic** [comment]
Hi @gaocegege, changed the subcategory name please merge this PR

**Comment by rahul-katonic** [comment]
Hi @gaocegege, any update for us?

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Pull request merged**